doc/nrf/known_issues.rst

```diff
@@ -153,7 +154,7 @@ CIA-661: Asset Tracker v2 application configured for LwM2M cannot be built for t
 Serial LTE Modem
```

Comment:
New entry for the Asset Tracker v2:

CIA-845: The application cannot be built with `overlay-carrier.lib` (carrier library) enabled for Thingy:91
	       Building with :ref:`liblwm2m_carrier_readme` library enabled for Thingy:91 will result in a FLASH overflow and a build error.

Reply:
@simensrostad , no workaround for this one?

Reply:
No, no good workaround atm I'm afraid.